src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,13 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+<<<<<<< Updated upstream
+=======
+            <class name="Homework16"/>
+            <class name="Homework17"/>
+            <class name="Homework18"/>
+
+>>>>>>> Stashed changes
         </classes>
     </test>
 </suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223759300-12d9395e-6775-4c63-8ffc-5a38121afec8.png)